### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by jayeshkattar** [GirlScriptSummerOfCode/Kranti#2]
Build a wire frame.

Guidelines:
Red! Other bright colors like blue, purple, and green are also good. No pink please.
Clean, simple, bold design. 
Prominent pictures on every page (photos in “Pics” folder, choose what you think looks best). 
No excessive info (We are WAY too wordy! Ha!)
Sites we like the look of:
http://www.teachforindia.org/
http://www.sanlaap.org/
https://www.thefourfountainsspa.in/

Content Here: https://drive.google.com/drive/folders/0BwpCIeAR0xFcTVlXc1FvQXZvcHM?usp=sharing

**Comment by adi44** [comment]
I want to work on this issue

**Comment by sourabhpruthi10** [comment]
I also want to work on it actually @jayeshkattar if it is possible can you please divide the work if the work is large and if you don't mind @adi44
 otherwise @adi44 you can take up this no problem

**Comment by siddhishree** [comment]
i would like to work on this issue. if possible, i also prefer division of work @sourabh1031 @adi44

**Comment by adi44** [comment]
I have no issue @sourabh1031 @jayeshkattar @frost-queen

**Comment by aish51197** [comment]
i would like to work on the issue if the work is divided.@jayeshkattar

**Comment by lalitmee** [comment]
Hey @adi44, you have been assigned with other issue and we can't assign more than one issue to the same person. 

So @aish51197 is assigned for this issue. Hey @aish51197, please tell us that how much time you will take to complete this issue.

**Comment by AvaniVerma** [comment]
@aish51197 If you could please reply a yes or no, we'll know that you've seen it. If you fail to do so by 10PM tonight we'll assign it to someone else.

**Comment by sourabhpruthi10** [comment]
@AvaniVerma @jayeshkattar can i work now on this issue now ?

**Comment by AvaniVerma** [comment]
@sourabh1031 Sure. Go ahead. Also please tell us how much time will you take to complete this issue.

**Comment by sourabh1031** [comment]
@AvaniVerma thanks for assigning actually i want to ask some of the prerequisites that you can provide me so that it would be easy for me to build it up. i will submit the pr by tommorow night or next early morning

**Comment by sourabhpruthi10** [comment]
it will take me two to three days to complete the task.

**Comment by troublemagnet** [comment]
Since this issue is about design and not code, it's a good idea to consider multiple opinions and ideas about how the website should look like. We can review multiple ideas from you guys and possibly mix and match all of them to decide the final look of the website. Hence whoever is interested, please feel free to submit your ideas, don't worry about the assignment. But we'd ideally like to close the design this week, hence please submit your ideas by Sunday morning. @frost-queen @adi44 @sourabh1031 @aish51197 

PS - Do not worry about points, your efforts will be rewarded.

**Comment by shreyateeza** [comment]
@troublemagnet  Can I also work on this issue? I will submit it in a few days. Please reply as soon as possible.

**Comment by troublemagnet** [comment]
@shreyateeza yes, go ahead. Try and submit by sunday though.

**Comment by shreyateeza** [comment]
@troublemagnet Can I create the website using wordpress straightaway? Or you want only the wireframes(using any software) for now?

**Comment by shreyateeza** [comment]
@jayeshkattar  Can I create the website using wordpress straightaway as mentioned in the google doc? Or you want only the wireframes(using any software) for now?

**Comment by adi44** [comment]
Pls give me a tab for creating a webpage for it m

**Comment by lalitmee** [comment]
Hey @shreyateeza, please select any tab on which you want to work or have any idea for implementation. We want you to do some research on the tabs so that we can select a good template for tabs. :+1: :innocent:

**Comment by muskan21jain** [comment]
I want to work on that. Can i go with this issue?

**Comment by shreyateeza** [comment]
@lalitmee Got you!  I am working on the Homepage!

**Comment by jayeshkattar** [comment]
@94151  you can work on this.

Guys whoever is working on this please create your own repository. Push the code to it and comment the repo link by tomorrow morning. We will be finalizing it tomorrow or will add few points where the design could be improved.

**Comment by shreyateeza** [comment]
@jayeshkattar Wouldn't it be better if issue #3 gets completed first then we can just fork this repo and do the changes?
Because in creating ones repo means creating the project from scratch would take pretty much time and may vary from one person to other.

**Comment by troublemagnet** [comment]
@shreyateeza yeah, you don't need to create anything from scratch. Just make a PR with your mocks for the home page.

**Comment by jayeshkattar** [comment]
@shreyateeza it's not necessary for you to code. Submit the wire frames. in .psd or .jpg

@all please note.

**Comment by vaarigupta** [comment]
I would like to work upon this issue .Is this still available? @jayeshkattar

**Comment by shreyateeza** [comment]
@vaarigupta you can work on this issue. You have to comment here the tab for which you are making the wireframe.
